bookkeeper-server/src/test/java/org/apache/bookkeeper/replication/TestReplicationWorker.java

```diff
@@ -502,10 +506,11 @@ public void testBookiesNotAvailableScenarioForReplicationWorker() throws Excepti
                 @Override
                 public void run() {
                     try {
-                        Thread.sleep(4000);
+                        Thread.sleep(3000);
                         isBookieRestarted.set(true);
+                        Thread.sleep(500);
                         /*
-                         * after sleeping for 4000 msecs, restart one of the
+                         * after sleeping for 3000 msecs, restart one of the
```

Comment:
do we have a better to wait ?
Just I don't want to see another flaky test on CI

not blocker, the test is already using Thraed.sleep

Reply:
basically here we are starting Bookie after sometime (3 secs), till then the ledger is expected to remain underreplicated. So we should be good with this sleep, since we are not depending on this sleep for an event to happen.

Reply:
ok